chaincode/tuple_aggregate.go

```diff
@@ -64,7 +64,15 @@ func (tuple *Aggregatetuple) SetFromInput(db *LedgerDB, inp inputAggregatetuple)
 func (tuple *Aggregatetuple) SetFromParents(db *LedgerDB, inModels []string) error {
 	var parentStatuses []string
 	inModelKeys := tuple.InModelKeys
-	permissions, err := NewPermissions(db, OpenPermissions)
+	owner, err := GetTxCreator(db.cc)
+	if err != nil {
+		return errors.BadRequest(err, "could not transaction creator")
+	}
+
+	authorizedIdsMap := map[string]bool{
```

Comment:
You don't seem to use the flag, only the key:
```suggestion
	authorizedIds := []string{}
```

Reply:
I'm using a map here to find _unique_ msp ids. I wish there was a less verbose/convoluted way to do this in Go, but there isn't!

Reply:
Ha good point 👍 You can then use a `map[string]struct{}` to lower the memory footprint. (this is yak shaving, feel free to dismiss this comment)